docs/reference/run.md

```diff
@@ -844,6 +846,16 @@ weights of the two containers.
 > **Note:** The blkio weight setting is only available for direct IO. Buffered IO
 > is not currently supported.
 
+The `--read-bps-device` flag can specify upper limit on READ rate from the device.
+For example, the command create one container and specify upper limit on READ rate from /dev/sda:
+
+    $ docker run -ti --name c1 --read-bps-device /dev/sda:300 ubuntu:14.04 /bin/bash
```

Comment:
this should use the "to human units" so for example `--read-bps-device /dev/sda:1mb` much like how the memory cgroups currently work